packages/diffast-etc/diffast-etc.0.1/opam

```diff
@@ -0,0 +1,45 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Diff/AST etc files"
+description: "Diff/AST etc files"
```

Comment:
Can you be a bit more descriptive? As is, it is hard to understand what this package is about